src/main/cc/proxy_client/proxy_client.cc

```diff
@@ -200,8 +204,39 @@ int ComputeInputs(int argc, char** argv, const char** env, const string& cwd, co
       // Fall back on computing from the command, but warn.
       cerr << cmd_id << "> Include processor did not return results, computing from args\n";
     }
+  } else if (string(argv[4]).find("header-abi-dumper") != std::string::npos) {
+    use_args_inputs = true;
+    *is_compile = true;
+    char **new_argv = (char**)malloc((argc+1) * sizeof *new_argv);
```

Comment:
It seems like the convention in this file that Ola is following is to use vector<char *> and then eventually const_cast() it. Feel free to ignore for now though if it works, we can refactor later.